### ISSUE TITLE
Improve contrast rate of selected Tree items by removing the doubled highlight

### ISSUE THREAD

**Issue by duianto** [godotengine/godot-proposals#3717]
### Describe the project you are working on

Not project specific.

### Describe the problem or limitation you are having in your project

Not project specific.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Improve the contrast of the selected line in the Error panel.

### Reproduction steps
- Open an existing or create a new project, for example: `New Game Project`
- Add a `Node2D`
- Add a script to `Node2D`
- In the `_ready()` function, replace:
```
	pass # Replace with function body.
```
with:
```
	push_warning("Lorem ipsum dolor sit amet, consectetur adipiscing elit, ")
	push_error("sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.")
```
- Press `F6` to run the scene
- In the window `Save scene before running...`
Click the `Save` button
- Close the window: `New Game Project (DEBUG)`
- At the bottom of Godot, click on `Debugger`
- Click on the tab: `Errors`
- Click on the red text: "sed do eiusmod"...

### Observed
The bright selected line highlight makes it difficult to read the red text.

![godot_error](https://user-images.githubusercontent.com/13420573/147389069-810941c4-07d7-4ee1-9089-8f251d8629b1.png)

The current workaround, to read the error text,
is to select another line in the error panel,
or to left click on the selected line, which hides the selection.

### Expected
The colors should be changed, to make the text more readable.

### Notes
It's slightly easier to read a selected warning text.
But it's contrast could probably also be improved.

![godot_warning](https://user-images.githubusercontent.com/13420573/147389071-5ae982d0-adb1-4468-99b6-dd08e0c1251e.png)

---

It might be helpful to check the contrast, on the page: https://accessible-colors.com/
when choosing the new colors.

If the current colors were color picked correctly,
then the accessibility checker page says that.

### Error text
The red error text color: `#FF786B`
The selected lines background color: `#747882`

The page says that the contrast ratio is: 1.72
The required contrast ratio for `AA` compliance is: 4.5

![accessibility_contrast_error](https://user-images.githubusercontent.com/13420573/147389079-00aeb46d-be30-492f-a91a-185e9cba9685.png)

### Warning text
The yellow warning text color: `#FFDE66`
The selected lines background color: `#747882`

The page says that the contrast ratio is: 3.34
The required contrast ratio for `AA` compliance is: 4.5

![accessibility_contrast_warning](https://user-images.githubusercontent.com/13420573/147389082-6a9ceb5e-98a4-4a3d-b500-7a7b17f17dd9.png)

### If this enhancement will not be used often, can it be worked around with a few lines of script?

This is used every time a line is selected in the Errors panel.

### Is there a reason why this should be core and not an add-on in the asset library?

This should be part of the core.

**Comment by Calinou** [comment]
This is due to the Tree highlight stylebox appearing twice for selected items. We should aim to fix that instead, as it will automatically resolve the issue with text contrast of highlighted items.

**Comment by duianto** [comment]
The double highlight might have been fixed in v4.0.beta5.official [89a33d28f]
It's at least easier to read the selected text now:

![Godot_v4 0-beta5_win64_0qBfQ2m9oM](https://user-images.githubusercontent.com/13420573/203310076-f6311d07-5363-49ba-b228-1595a0d788b6.png)

![Godot_v4 0-beta5_win64_XwGq3D5OkR](https://user-images.githubusercontent.com/13420573/203310071-5d15aaf3-c1a3-4828-b37f-9e45d26eb9c5.png)

**Comment by Calinou** [comment]
I can confirm it's fixed as of 4.0.beta5.
